api/tests/integration/infrastructure/repositories/private-certificate-repository_test.js

```diff
@@ -38,7 +38,7 @@ describe('Integration | Infrastructure | Repository | Private Certificate', func
   describe('#get', function () {
     it('should throw a NotFoundError when private certificate does not exist', async function () {
       // when
-      const error = await catchErr(privateCertificateRepository.get)(123);
+      const error = await catchErr(privateCertificateRepository.get)(123, 'fr');
```

Comment:
Est-ce qu'on ne s'attends pas à: 
```suggestion
      const error = await catchErr(privateCertificateRepository.get)(123, {locale: 'fr'});
```